experiments/openebs/openebs-pool-network-delay/openebs_pool_network_delay_k8s_job.yml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: openebs-pool-network-delay
+  namespace: litmus
+data:
+  parameters.yml: |
```

Comment:
please don't provide any data inside this

Reply:
have the same content as in the docs. Or recommended to skip it